.github/workflows/changelog.yml

```diff
@@ -163,6 +163,8 @@ jobs:
 
             // Nothing to complain about, so delete any existing comment
             await createOrUpdateChangelogComment("", true);
+      - name: Checkout changie config
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
I think you would need to use 
```
with:
  ref: github.head_ref
```

here. Since this is a `pull_request_target` workflow the ref that would be used by default is the target branch of the PR, so in our case e.g. main and not the branch under test (see comment in the beginning of this file).

Another way to solve this would be to create a different workflow for this validation altogether

Reply:
I see, fair point and good catch - thanks, I'll make the change